internal/cmd/gocdk/internal/static/_assets/launchers/ecs.tf

```diff
@@ -0,0 +1,170 @@
+# Create an ECR Docker repository.
+# The name used here will be part of the deployed Docker image name.
+resource "aws_ecr_repository" "default" {
+  name = "{{.ProjectName}}"
+}
+
+# ECS Cluster
+
+resource "aws_ecs_cluster" "default" {
+  name = random_id.cluster_name.hex
```

Comment:
I have been using `local.gocdk_random_name` for random string everywhere, so that it's always the same...? It's currently using `gocdk-` as the prefix instead of `{{.ProjectName}}`, but we could fix that independently.

Reply:
It would be nice to have the include the project name instead of `gocdk`, but I agree, that's a separate issue.